sensors/tests/test_pressure_driver.cpp

```diff
@@ -145,26 +146,17 @@ SCENARIO("Read pressure sensor values") {
                                     0x0}};
                 driver.handle_response(sensor_response);
                 THEN(
-                    "the handle_message function sends the correct data via "
-                    "the CAN bus, and the sync pin is set") {
-                    can::message_writer_task::TaskMessage can_msg{};
+                    "no data is sent via the CAN bus, and the sync pin is "
+                    "set") {
+                    REQUIRE(can_queue.get_size() == 0);
 
-                    can_queue.try_read(&can_msg);
-                    auto response_msg =
-                        std::get<can::messages::ReadFromSensorResponse>(
-                            can_msg.message);
-                    float check_data =
-                        fixed_point_to_float(response_msg.sensor_data, 16);
-                    float expected = 33.53677;
-                    REQUIRE(check_data == Approx(expected));
                     REQUIRE(hardware.get_sync_state_mock() == true);
                 }
             }
         }
     }
-    GIVEN(
-        "Pressure read with SensorMode = POLLING, and output binding = "
-        "report") {
+    GIVEN("Pressure read with output binding = report") {
```

Comment:
Can we also do a multi-tier call test? What I mean by that would be, in the same `GIVEN` call:

1. Do a limited poll call
2. Do a single read
3. Do a continuous poll call
4. Do another single read call